### PR TITLE
163 Change to a simpler checkbox

### PR DESCRIPTION
**Pull request by SergioAAI**
# Description

Implements/fixes [SOEN-163](https://soen390team8.atlassian.net/browse/SOEN-163)

- Changed it to a clearer checkbox for the doctor modal

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Unit tested

# Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** original screenshot ** | ** updated screenshot ** |

Checkmark2

### CODE REVIEW

**Review by KelvinTrang — APPROVED** [review]
nice changes

**Review by KelvinTrang — APPROVED** [review]

File: frontend/components/doctor/patient-info-modal-swiper.tsx [hunk 1]
```diff
@@ -31,9 +32,27 @@ const PatientInfoModalSwiper = ({ patient, onMutate }: AppProps) => {
...
+                    title: "Success",
+                    description: "This status has been set as reviewed",
+                    status: "success",
+                    duration: 3000,
+                    isClosable: true,
+                });
+                onMutate();
+            })
+            .catch(err => {
+                console.log(err);
```
Comment (NicholasYip) [inline frontend/components/doctor/patient-info-modal-swiper.tsx:47]: should we really log the error? isn't a modal enough?

**Review by NicholasYip — APPROVED** [review]
small comment, rest looks good

File: frontend/components/doctor/patient-info-modal-swiper.tsx (hunk 1 above)
Comment (SergioAAI) [reply frontend/components/doctor/patient-info-modal-swiper.tsx:47]: its to replicate the other behaviors in the modal , but it can be removed

**Review by amqkv — APPROVED** [review]
Looks good!

**Pull request merged**